apps/assistants/views.py

```diff
@@ -115,8 +155,12 @@ class DeleteOpenAiAssistant(LoginAndTeamRequiredMixin, View, PermissionRequiredM
     @transaction.atomic()
     def delete(self, request, team_slug: str, pk: int):
         assistant = get_object_or_404(OpenAiAssistant, team=request.team, pk=pk)
+        try:
+            delete_openai_assistant(assistant)
+        except OpenAiSyncError as e:
+            messages.error(request, f"Error deleting assistant from OpenAI: {e}")
+            return HttpResponse(status=500)
```

Comment:
Just confirming this wouldn't crash the page?

Reply:
no, it just displays the error message